deployment/deployment.json

```diff
@@ -7522,7 +7522,7 @@
           }
         },
         "files": {
-          "template": "polygon.bridge.template.yaml"
+          "template": "polygon-root.template.yaml"
```

Comment:
The naming here is a little unconventional. Can you name it similar to `polygon.mainnet.template.yaml`